sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Bugs Fixed
 - Fixed \[BUG\] BlobContainerClient(connectionString, blobContainerName, options) ctor to set clientSideEncryptionOptions #44623
+- Fixed \[BUG\] Make it possible to set etag when creating BlobDownloadDetails for unit testing #42009
```

Comment:
@amnguye, should we include this one in the changelog, since it is so minor?

Reply:
Let's not include in the changelog, since the change is small and when adding these mock constructors we haven't needed to update the changelog.

Reply:
Okay, removed Changelog